recipes/threadpool/all/conanfile.py

```diff
@@ -0,0 +1,35 @@
+import os
+from conans import ConanFile, tools
+
+class threadpoolConan(ConanFile):
+    name = "threadpool"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/progschj/ThreadPool"
+    description = "A simple C++11 Thread Pool implementation."
+    license = "Zlib"
+    topics = ("C++11", "Thread", "Pool", "threadpool", "conan")
+    settings = "os"
+    no_copy_source = True
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        listdir = os.listdir()
+        extracted_dir = [i for i in listdir if "ThreadPool" in i][0]
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def package(self):
+        self.copy(pattern="COPYING", src=self._source_subfolder, dst="licenses", )
+        self.copy(pattern="*.h", src=self._source_subfolder,  dst=os.path.join("include", "threadpool"))
```

Comment:
```suggestion
        self.copy(pattern="*.h", src=self._source_subfolder,  dst=os.path.join("include", "ThreadPool"))
```
to be consistent with header?

Reply:
HI Chris, thanks for the feedback.
I pushed a commit with this change instead of commiting yours since this change also means changing the `include` in the test package.